Automattic/jetpack
# Removes the WooCommerce installation menu item from WordPress.com

Fixes https://github.com/Automattic/dotcom-forge/issues/5236

## Proposed changes:

* Removes the WooCommerce installation menu item from WordPress.com

### Other information:

- [x] Removed existing unittest for these calls

## Jetpack product discussion
TBD / non jetpack

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Simple sites: (in your sandbox ~/public_html) `bin/jetpack-downloader test jetpack update/remove-woocommerce-installation-menu`
* Atomic sites, from local to a woatest blog, `jetpack rsync jetpack 1541434@woatest112.wpcomstaging.com:~/htdocs/wp-content/plugins`
* The WooCommerce menu item should be removed from both simple sites and your atomic test site.
